### ISSUE TITLE
Timestamp values in actuators are serialized as integers instead of date-time strings

### ISSUE THREAD

**Issue by deepthidevaki** [camunda/zeebe#17347]
**Describe the bug**

In 8.4, timestamp values were mapped to a string of format `2020-01-01T00:00Z`. But in 8.5.0 they are mapped to an integer. For example for the cluster topology endpoint:
````
     "state": "ACTIVE",
      "version": 0,
      "lastUpdatedAt": -62167219200,
      "partitions": [
````

This is not confirming to the openAPI spec which specifies a `date-time` format. 

**Impact**
QA & E2E tests fail because the chaos worker fails to parse the integer format.
Backup automation may break if it depends on the string format.

**To Reproduce**

Start a cluster on 8.5.0 and run following query
```
curl --request GET "http://localhost:9600/actuator/cluster" | jq
```

**Expected behavior**

`lastUpdatedTime` and other timestamps are serialized to a string date-time format.

**Environment:**
- Zeebe Version: 8.5.0

**Comment by deepthidevaki** [comment]
@npepinpe @tmetzke I assume some config you might have introduced for the rest api might have conflicted with this. Any idea?

**Comment by deepthidevaki** [comment]
Note that medic-14 benchmark also respond with the expected date-time format. So it must be some changes introduced after that.

**Comment by npepinpe** [comment]
Yes, sorry, it's the JacksonConfig configuration in dist. We now configure it for the main server, but the management server also uses it. Sorry

**Comment by npepinpe** [comment]
I thought it might cause issues, but no tests were failing...but we also merged last minute, so we didn't run the e2e tests =/

**Comment by deepthidevaki** [comment]
Do you mean this change? https://github.com/camunda/zeebe/commit/046014fd6175c5d97b317f7556063d85ac291767 But this should serialize it in date-time format, right?

**Comment by deepthidevaki** [comment]
> but no tests were failing

Will check it. May be the tests are not covering full serialization and deserialization path.

**Comment by npepinpe** [comment]
I assume it's that change. I suspect it's because we overwrite the default Jackson config =/

**Comment by npepinpe** [comment]
note that when we merge operate, which also has its own jackson config, this will likely happen again

**Comment by oleschoenburg** [comment]
I've updated the description because this bug affects all actuators, not just the scaling endpoint. Most importantly, the backup endpoints are affected too which may break any backup automation that depends on string values and can't deal with integers instead.

**Comment by deepthidevaki** [comment]
@oleschoenburg The issue can be closed, right? Why was it not closed automatically after PR merge?

**Comment by tmetzke** [comment]
> @oleschoenburg The issue can be closed, right? Why was it not closed automatically after PR merge?

Issue and PR weren't linked by GitHub automatically.

**Comment by oleschoenburg** [comment]
> Issue and PR weren't linked by GitHub automatically.

How so?

As far as I understand the issue wasn't closed because the PR to main only said "relates to" which doesn't close automatically.

**Comment by tmetzke** [comment]
> > Issue and PR weren't linked by GitHub automatically.
> 
> How so?
> 
> As far as I understand the issue wasn't closed because the PR to main only said "relates to" which doesn't close automatically.

The original PR to fix this on `stable/8.5` actually used "closes" but since it wasn't merged to `main` GitHub doesn't link this automatically, I guess? Yes, the backported issue that fixes this on `main` just used "relates to" which doesn't close the related issue 👍
